<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailboxWindow</class>
 <widget class="QMainWindow" name="MailboxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Meat4Eat - Mailbox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/application.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MailboxWindow {
  border: 1px solid rgb(151,167, 187);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
  background: transparent;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayoutMain">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widgetMain" native="true">
        <property name="styleSheet">
         <string notr="true">#widgetMain {
 background-color: white;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,2,0">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetHead" native="true">
             <property name="cursor">
              <cursorShape>SizeAllCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">#widgetHead {
  background-color: rgb(28,52,62);
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <widget class="QLabel" name="labelHeadText">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Roboto Black</family>
                  <pointsize>10</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLabel#labelHeadText {
  color: rgb(181,197, 217);;
}</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Mailbox&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
                <property name="margin">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <layout class="QHBoxLayout" name="topBar">
                  <property name="spacing">
                   <number>3</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonMimimize">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-minimize.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonMaximize">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-maximize.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonClose">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="focusPolicy">
                     <enum>Qt::NoFocus</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-close.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetMyMails" native="true">
               <property name="minimumSize">
                <size>
                 <width>294</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#widgetMyMails {
  background-color: rgb(28,52,62);
}</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10" stretch="0,0">
                <property name="bottomMargin">
                 <number>2</number>
                </property>
                <item>
                 <widget class="QWidget" name="widgetMailItemsHead" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>270</width>
                    <height>30</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
	  color: rgb(181,197, 217);;
}
QLabel {
	  color: rgb(181,197, 217);;
}</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_13">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_11">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="leftMargin">
                      <number>9</number>
                     </property>
                     <property name="rightMargin">
                      <number>9</number>
                     </property>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QLabel" name="labelHead">
                         <property name="minimumSize">
                          <size>
                           <width>0</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>16777215</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <family>Roboto</family>
                           <pointsize>11</pointsize>
                           <weight>75</weight>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>Your Mails</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="horizontalSpacer_4">
                         <property name="orientation">
                          <enum>Qt::Horizontal</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>10</width>
                           <height>20</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButtonNewMail">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="toolTip">
                          <string>Write a new mail</string>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 16px;
  border-image: url(:/icon-newevent.png);
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 16px;
}</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_2">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>2</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>2</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QLabel {
  background-color: rgb(151,167, 187);
}</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <widget class="QPushButton" name="pushButtonPrev">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="toolTip">
                          <string>Previous mails</string>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 16px;
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 16px;
}</string>
                         </property>
                         <property name="text">
                          <string>&lt;-</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="labelMailRange">
                         <property name="text">
                          <string>1-10</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButtonNext">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="toolTip">
                          <string>Next mails</string>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">QPushButton {
  background-color: transparent;
  border-radius: 16px;
}
QPushButton:hover {
  background-color: rgb(50,82,95);
  border-radius: 16px;
}</string>
                         </property>
                         <property name="text">
                          <string>-&gt;</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="minimumSize">
                        <size>
                         <width>0</width>
                         <height>2</height>
                        </size>
                       </property>
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>2</height>
                        </size>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QLabel {
  background-color: rgb(151,167, 187);
}</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widgetMailItems" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#widgetMailItems {
  background-color: rgb(28,52,62);
  border: 0;
}</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <property name="leftMargin">
                    <number>9</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widgetClientArea" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="sizeConstraint">
                 <enum>QLayout::SetMinAndMaxSize</enum>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="widgetFoot" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#widgetFoot {
 border: 1px solid rgb(28,52,62);
 background-color: rgb(28,52,62);
}</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="m4e::common::ButtonResize" name="pushButtonResizer">
                  <property name="minimumSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>26</width>
                    <height>26</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>SizeFDiagCursor</cursorShape>
                  </property>
                  <property name="mouseTracking">
                   <bool>true</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QPushButton {
  padding: 0;
  border: 1px solid rgb(131,147, 167);
  border-image: url(:/icon-resize.png);
}
QPushButton:hover {
  border: 5px solid white;
  border-radius:  4px;
}</string>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>m4e::common::ButtonResize</class>
   <extends>QPushButton</extends>
   <header location="global">common/btnresize.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonClose</tabstop>
  <tabstop>pushButtonMimimize</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>MailboxWindow</receiver>
   <slot>onBtnCloseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMimimize</sender>
   <signal>clicked()</signal>
   <receiver>MailboxWindow</receiver>
   <slot>onBtnMinimizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMaximize</sender>
   <signal>clicked()</signal>
   <receiver>MailboxWindow</receiver>
   <slot>onBtnMaximizeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNewMail</sender>
   <signal>clicked()</signal>
   <receiver>MailboxWindow</receiver>
   <slot>onBtnNewMailClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnCloseClicked()</slot>
  <slot>onBtnMinimizeClicked()</slot>
  <slot>onBtnResizeReleased()</slot>
  <slot>onBtnMaximizeClicked()</slot>
  <slot>onBtnNewMailClicked()</slot>
  <slot>onBtnNextClicked()</slot>
  <slot>onBtnPrevClicked()</slot>
 </slots>
</ui>
